components/widgets/menu/menu.jsx

```diff
@@ -21,6 +21,48 @@ export default class Menu extends React.PureComponent {
         this.node = React.createRef();
     }
 
+    hideUnneededDividers = () => {
+        if (this.node.current === null) {
+            return;
+        }
+
+        const children = Object.values(this.node.current.children).slice(0, this.node.current.children.length);
+
+        // Hiding formers dividers and duplicated ones
```

Comment:
typo. "formers". I suppose you meant "former" as in previous? Or "form" ?

Reply:
Yes, I meant the group of dividers consecutively placed at the beginning of the list of elements.

Reply:
Changed with a more clear text